Modules/System/Upgrade Tags/src/UpgradeTag.Codeunit.al

```diff
@@ -85,7 +104,7 @@ codeunit 9999 "Upgrade Tag"
     /// List of upgrade tags that should be inserted if they do not exist.
     /// </param>
     [IntegrationEvent(false, false)]
-    internal procedure OnGetPerDatabaseUpgradeTags(var PerDatabaseUpgradeTags: List of [Code[250]])
+    procedure OnGetPerDatabaseUpgradeTags(var PerDatabaseUpgradeTags: List of [Code[250]])
```

Comment:
Same comment here. It would be nice to get a public method that calls the internal implementation.

Reply:
procedure GetPerDatabaseUpgradeTags added